app/controllers/activities_controller.rb

```diff
@@ -13,6 +14,7 @@ def show
   end
 
   def create
+    # byebug
```

Comment:
I would delete this instead of leaving it as a comment.  But I'm still approving the PR.